### DISCUSSION TITLE
Where are the sensor settings in the web ui gone?

### DISCUSSION THREAD

**Discussion by rademakeronline** [ESPresense/ESPresense#296] [Support]
Hi all,

first, I want to say that I like this project very much and the active development. But since one of the last updates, the settings for the brightness and DHT sensors aren't available in the web ui anymore. Unfortunately, I'm not able to find any information about this topic. 

Can anybody tell me, where they did go? Or which was the last version, with the old web ui settings? And if it's possible to do an OTA firmware downgrade?

Thanks in advance
Christian

**Comment by DTTerastar** [comment]
There is now a sensors firmware flavor that you can install to get all the sensors goodness.  It was separated to conserve space.  Plug it into usb, reflash and from that point on it'll auto update using the sensors flavor.

**Reply by rademakeronline** [reply]
Thanks for that reply. I did exactly what you mentioned. However, I don't see any sensor settings but motion and radar.

> https://espresense.com/releases/latest/download/esp32-sensors.bin
This is the right one?
